### ISSUE TITLE
Warning: Package Arpack does not have Libdl in its dependencies

### ISSUE THREAD

**Issue by sbromberger** [JuliaLinearAlgebra/Arpack.jl#24]
I downloaded Julia -rc3 binaries and did a `]add Arpack` and a `]build Arpack`. Now, when I try `using Arpack` at the REPL, I get the following:
```
julia> using Arpack
[ Info: Recompiling stale cache file /Users/bromberger1/.julia/compiled/v0.7/Arpack/X5VZL.ji for Arpack [7d9fca2a-8960-54d3-9f78-7d1dccf2cb97]
┌ Warning: __precompile__() is now the default
│   caller = top-level scope at none:0
└ @ Core none:0
┌ Warning: Package Arpack does not have Libdl in its dependencies:
│ - If you have Arpack checked out for development and have
│   added Libdl as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with Arpack
└ Loading Libdl into Arpack from project dependency, future warnings for Arpack are suppressed.
```

`eigs` seems to work, but every time I do a `using Arpack` in the REPL, it tries to recompile.

**Comment by sbromberger** [comment]
This is on
```
julia> versioninfo()
Julia Version 0.7.0-rc3.0
Commit 5f101bb222 (2018-08-06 23:10 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin14.5.0)
  CPU: Intel(R) Core(TM) i5-6267U CPU @ 2.90GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, skylake)
```

**Comment by simonbyrne** [comment]
Closed by #28.

**Comment by sbromberger** [comment]
Thank you!
